tinyagentos/cluster/manager.py

```
@@ -598,6 +619,20 @@ async def claim_lease(
                 )
                 return None
 
+            # Enforce lease cap per worker to prevent DoS (S2-24)
+            # Count existing leases for this worker
+            worker_lease_count = 0
+            for lid, lease in self._leases.items():
```

Ruff (0.16.3)

[warning] Loop control variable `lid` not used within loop body

Rename unused `lid` to `_lid`

(B007)